### ISSUE TITLE
Minitwit crashed again - But did recover

### ISSUE THREAD

**Issue by Thomsen96** [DevelOpsITU/MiniTwit#86]
From looking at the logs this morning it was visible that the minitwit application had crashed again. 

See below: Status is only 2 hours ago. That means that it was re created. 
![billede](https://user-images.githubusercontent.com/32035863/156529511-2039dc98-f1a1-453c-b716-2ea52674d34e.png)

Logs from morning at 06:19.
![billede](https://user-images.githubusercontent.com/32035863/156529337-77d6df9f-0d31-43e4-b46b-e4a92fb8912c.png)

We see a big twit being posted and causing the crash.

The docker-compose sees that the program crashes and restarts it.

**Comment by ValarMarkhulis** [comment]
I tried to run the simulator on the dev branch this morning, where I tried to send a similar big message of 8000 chars. That did not seem to crash my local instance.

But when I let it keep running for some time I got an error message saying:
"2022/03/03 09:42:17 unable to open database file: too many open files"
And after looking up the close() method for our sql driver, I noticed that according to their documentation, we are using the driver wrong. It says that it is meant to be a long-lived and shared connection to the database.
![image](https://user-images.githubusercontent.com/11947367/156531180-b0c240c2-07ee-4274-992c-93efb25dc35c.png)

Right now we are calling ConnectDb() every time we want to do a sql stmt, which creates another db connection and does not reuse our existing connection. This is also clearly stated in the documentation:
![image](https://user-images.githubusercontent.com/11947367/156531836-31ae2e99-e477-4265-a673-8abb177972f0.png)

I'm not sure if this is the reason of our crash, but it shows that we are for certain not using the small local sqlite database in the most efficient way possible. Therefor @Thomsen96 and I will try to finish up #37, so we can use [the deployed postgres server](https://github.com/DevelOpsITU/ServerDeployment/issues/9)
